src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -338,8 +340,56 @@ private bool CanInstallPackage()
                 return false;
             }
 
+            return true;
+        }
+
+        public string ErrorMessage
+        {
+            get { return _errorMessage; }
+            set { Set(ref _errorMessage, value); }
+        }
+
+        public bool IsLibraryInstallationStateValid()
```

Comment:
>IsLibraryInstallationStateValid [](start = 20, length = 31)

Should this be made async? I only see the one call from an async method in this class.

Reply:
ok, I now see the reference outside this class. Still wonder whether it should be async.

---
In reply to: [201159303](https://github.com/aspnet/LibraryManager/pull/293#discussion_r201159303) [](ancestors = 201159303)

Reply:
Made this async in iteration 7

---
In reply to: [201161101](https://github.com/aspnet/LibraryManager/pull/293#discussion_r201161101) [](ancestors = 201161101,201159303)